CyReP/tcps/TcpsId.c

```diff
@@ -459,19 +460,21 @@ GetClaim(
     CborError err = pDecodeTCPSIdentity(Id, IdSize, &claimSet);
     if (err != CborNoError)
     {
+        status = RIOT_FAILURE;
         goto Cleanup;
     }
 
     claimIndex = pFindAssertion(Name, claimSet.AssertionArray, claimSet.Used);
     if (claimIndex == -1)
     {
+        status = RIOT_FAILURE;
```

Comment:
Whoever wrote this function should have left comments, but I'm wondering if the intent was that not finding the named assertion was not a failure? Does GetClaim()'s contract require that the claim exist? Guessing at this point there might not be too many callers, but might be nice to be able to tell the difference between an error and a missing value.

Reply:
The intent was to fail on claim_not_found, and the behavior has not changed with this change. Agreed that not found should not necessarily be a fatal failure. Will extend the interface.